subsys/partition_manager/ext_flash_mcuboot_secondary.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=y
```

Comment:
Is this file really needed for a single parameter ?

Reply:
See https://github.com/nrfconnect/sdk-nrf/pull/5439#discussion_r700126760